### ISSUE TITLE
hb-service and NODE_EXTRA_CA_CERTS

### ISSUE THREAD

**Issue by Jame-Stitel** [oznu/homebridge-config-ui-x#1021]
**Describe Your Problem:**
I need to set up a NODE_EXTRA_CA_CERTS environment variable for running homebridge, how can I do this with hb-service?

**Logs:**
```
[FibaroHC3] Error getting data from Home Center:  Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (_tls_wrap.js:1497:34)
    at TLSSocket.emit (events.js:315:20)
    at TLSSocket._finishInit (_tls_wrap.js:932:8)
    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:706:12) {
  code: 'UNABLE_TO_GET_ISSUER_CERT_LOCALLY'
}
```

**Comment by oznu** [comment]
Linux: `/etc/default/homebridge`

macOS: `/Library/LaunchDaemons/com.homebridge.server.plist`

**Comment by Jame-Stitel** [comment]
I added it there as ```NODE_EXTRA_CA_CERTS="/home/pi/Downloads/ca.crt"``` but still cannot control the accessories :/ although that error with UNABLE_TO_GET_ISSUER_LOCALLY is not showing anymore. But I added the certificate to other places as well, so didn't really know which helped with it. :D 

The thing is though, that I still cannot control the accessorries (Fibaro) when I set it to accept only https, when I set it to accept http, everything works... so it still is probably an issue with SSL certificate?
